crates/extensions/c8y_mapper_ext/src/operations/log_upload.rs

```diff
@@ -184,69 +170,49 @@ impl CumulocityConverter {
             Some(command.into_generic_command(&self.mqtt_schema)),
         ))
     }
+}
 
-    /// Resumes `log_upload` operation after required file was downloaded from
-    /// the File Transfer Service.
-    pub async fn handle_fts_log_download_result(
-        &mut self,
-        cmd_id: CmdId,
-        download_result: DownloadResult,
-        fts_download: FtsDownloadOperationData,
-    ) -> Result<Vec<MqttMessage>, ConversionError> {
-        let topic_id = fts_download.entity_topic_id;
-        let smartrest_topic = self.smartrest_publish_topic_for_entity(&topic_id)?;
-        let payload = fts_download.message.payload_str()?;
-        let response = &LogUploadCmdPayload::from_json(payload)?;
-
-        let download_response = match download_result {
-            Err(err) => {
-                let smartrest_error = fail_operation(
-                    CumulocitySupportedOperations::C8yLogFileRequest,
-                    &format!(
-                        "tedge-mapper-c8y failed to download log from file transfer service: {err}",
-                    ),
-                );
+impl CumulocityConverter {
```

Comment:
Minor: Same comment as in config_snaphot.rs. Better to restore this function and the next impl block to its former location to reduce the diff size, and to better highlight the changed bits.

Reply:
Indeed, the diff looks much easier to review. I'll remember not to shuffle the code next time for easier review.
fixed in b8bd269